chaos-common/src/main/java/io/openchaos/common/utils/PauseProcessUtil.java

```diff
@@ -39,9 +42,12 @@ public static void suspend(String node, String processName) throws Exception {
     public static void resume(String node, String processName) throws Exception {
 
         log.info("Resume node {} process {} .", node, processName);
-        String pid = SshUtil.execCommandWithArgsReturnStr(node, String.format("ps ax | grep -i '%s' | grep java | grep -v grep | awk '{print $1}'", processName)).trim();
-        if (!pid.isEmpty()) {
-            SshUtil.execCommand(node, String.format("kill -CONT %s", pid));
+        String pidList = SshUtil.execCommandWithArgsReturnStr(node, String.format("ps ax | grep -i '%s' | grep java | grep -v grep | awk '{print $1}'", processName)).trim();
```

Comment:
The command should not be hard coded. I will open this issue for the next improvement.